<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMyLogHandler</class>
 <widget class="QMainWindow" name="KMyLogHandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KMyLogHandler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="output">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="tofileButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>写入文本文件</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tofileEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>104</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="tobinaryButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>写入二进制文件</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tobinaryEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>104</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="fromfileBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fromfileButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>读取文本文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="frombinaryButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>读取二进制文件</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="frombinaryBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>日志内容：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
